packages/dotcom-ui-header/src/header.scss

```diff
@@ -34,7 +34,10 @@
 .o-header__drawer-search-term {
   width: 100%;
 }
-
+.o-header__top-wrapper .ft-header__ask-ft-button {
+  padding: 8px;
+  margin-left: -12px;
```

Comment:
**question:** Has this been tested in all devices sizes? I am not a fan of negative values for layout rules unless it is absolutely necessary?

Reply:
the new structure moves a little bit the ask ft button to the right but maybe is something that we can assume and avoid negative margin